Userland/Games/2048/Game.cpp

```diff
@@ -207,6 +218,56 @@ Game::MoveOutcome Game::attempt_move(Direction direction)
     return MoveOutcome::InvalidMove;
 }
 
+void Game::add_evil_tile()
+{
+    auto worst_row = 0;
+    auto worst_column = 0;
+    auto worst_value = 2;
```

Comment:
I don't know what the latest Serenity trends are, but back in my day we wouldn't use this much `auto`.

Which type do these variables get, by the way? `int`? `i32`? `size_t`?

(Naturally, they *should* be `size_t`.)

Reply:
Fair point, I'm too used to AAA.